ui/pages/confirmations/hooks/useRedesignedSendFlow.ts

```diff
@@ -11,13 +10,10 @@ export const useRedesignedSendFlow = () => {
   const { sendRedesign: sendRedesignFeatureFlag } = useSelector(
     getRemoteFeatureFlags,
   );
-  const isMultichainAccountsState2Enabled = useSelector(
-    getIsMultichainAccountsState2Enabled,
-  );
   const { enabled: isSendRedesignEnabled } = (sendRedesignFeatureFlag ??
     {}) as SendRedesignFeatureFlag;
```

Comment:
As we talked earlier - can we make this work like a true kill-switch

```suggestion
const { enabled: isSendRedesignEnabled = true } =
    (sendRedesignFeatureFlag ?? { enabled: true }) as SendRedesignFeatureFlag;
```

Reply:
That is resulting in lot of e2e tried here: https://github.com/MetaMask/metamask-extension/pull/36506

I want to do this change to see if it can fix the bug @bschorchit is getting.